python/glow/gwas/lin_reg.py

```diff
@@ -201,6 +197,7 @@ def _linear_regression_inner(genotype_pdf: pd.DataFrame, Y_state: YState,
     pvalues = 2 * stats.distributions.t.sf(np.abs(T), dof)
 
     del genotype_pdf[_VALUES_COLUMN_NAME]
+
```

Comment:
Can you revert the whitespace changes? It'll minimize unnecessary merge conflicts.